### DISCUSSION TITLE
Hotspot client not resolving names (DNS Issue)

### DISCUSSION THREAD

**Discussion by Back-Web** [Mygod/VPNHotspot#630] [Q&A] [answered]
Hotspot clients are using the VPN Hotspot's IP address "192.168.49.1" as default gateway and DNS server but when the VPN is established the Hotspot stop to resolve name. The client can ping 8.8.8.8 or any other IP throughput VPN.

VPNHotspot never stops accessing the Internet, whether the tunnel is established or not. The problem only occurs for the client, it seems that the VPNHotspot for resolving names "not forward the query throughput tunnel".

Does anyone know how to solve it?

VPN Hotspot 2.15.5-g
Android 8.1.0

**Comment by Mygod** [comment, marked as answer]
v2.15.5 is terribly outdated. Update to latest version.

EDIT: Android 8.1 is also no longer supported.

**Reply by Back-Web** [reply]
Tks!!!
